app/controllers/contact_details_controller.rb

```diff
@@ -1,3 +1,23 @@
 class ContactDetailsController < ApplicationController
-  include InheritAction
+  skip_before_action :doorkeeper_authorize!
```

Comment:
Skip authentiocation only for `show` method, in `update` authentication is required.